src/service/command/meme/tsureteike.ts

```diff
@@ -0,0 +1,32 @@
+import type { MemeTemplate } from '../../../model/meme-template.js';
+
+export const tsureteike: MemeTemplate<never, never> = {
+  commandNames: ['tsureteike', 'hunt'],
+  description: `「この中に〜はいるか 連れて行け」\nex.) \`!tsureteike プログラマ Rustは知っているか? ゲームですか? 錆のこと? 🦀\``,
```

Comment:
```suggestion
  description: '「この中に〜はいるか 連れて行け」\nex.) `!tsureteike プログラマ Rustは知っているか? ゲームですか? 錆のこと? 🦀`',
```
通常の文字列にすればいくつかのエスケープは不要かと